src/ui/components/CCDiceMenu.vue

```diff
@@ -323,12 +323,13 @@ export default Vue.extend({
   },
   watch: {
     menu() {
-      if (!this.menu || !this.autoroll) this.reset()
```

Comment:
This change allows for autorolled dice menus to keep their value even after closing. This helps keep the base die roll more consistent.